src/main/java/org/ossgang/commons/observable/DispatchingObservable.java

```diff
@@ -109,13 +110,23 @@ protected void dispatchException(Throwable newValue, Predicate<Set<SubscriptionO
         DISPATCHER_POOL.submit(() -> {
             try {
                 handler.accept(value);
+            } catch (UnhandledException e) {
+                uncaughtExceptionHandler.accept(e);
```

Comment:
should we also surround these 2 consumers by try-catch? With just print err in this case... :)

Reply:
Done. I've also added a null check in the setter, just in case someone tries to be stupid ;-)